src/product-and-services/index.md

```diff
@@ -3,30 +3,27 @@ layout: layouts/default.njk
 title: Products and Services
 ---
 
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6 float-left mr-2 mt-8">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M12 18v-5.25m0 0a6.01 6.01 0 001.5-.189m-1.5.189a6.01 6.01 0 01-1.5-.189m3.75 7.478a12.06 12.06 0 01-4.5 0m3.75 2.383a14.406 14.406 0 01-3 0M14.25 18v-.192c0-.983.658-1.823 1.508-2.316a7.5 7.5 0 10-7.517 0c.85.493 1.509 1.333 1.509 2.316V18" />
+</svg>
+
 ## Products
 
 #### [Convene]
 
-[Convene] Convene creates human connection through safe, accessible and reliable video conferencing.
-
-#### [Compensated]
-
-[Compensated] makes it easy for product and service developers to accept money from whichever payment processor they prefer with a low implementation or switching costs.
-
-#### [Catalog Choice]
-
-[Catalog Choice] helps people reduce their environmental footprint by opting out of catalog mailings. It is part of [The Story of Stuff project].
+An app that creates human connection through safe, accessible and reliable video conferencing.
```

Comment:
While we're here, let's update this blurb to remove the mention to videoconferencing, since that functionality no longer exists. The new text can be something like:

> ### Convene
> Your space on the web, fully configurable, developed and operated sustainably and in community. Current features include publishing content, selling and accepting payments.

Reply:
✅  https://github.com/zinc-collective/www.zinc.coop/pull/101/commits/ad7050df2a73f76e134ce4901035a57622b8b995

<img width="610" alt="Screenshot 2023-06-08 at 10 37 36 AM" src="https://github.com/zinc-collective/www.zinc.coop/assets/16140873/cf54bf50-5db1-4395-a8d3-f36c8d744bd2">

Reply:
Thank you!